src/static/css/content.less

```diff
@@ -414,48 +414,69 @@ figure {
 .minicon-scale-table {
   th,
   td {
-    padding: 0 5px;
-    border-left: 1px solid @gray-10;
+    padding: 10px 5px;
     background: transparent !important;
     color: @black !important;
     text-align: center;
 
-    &:first-child {
-        border: none;
+    &:not(:last-child) {
+      border-right: 1px solid @gray-10;
     }
   }
+
   th {
-    .u-webfont-demi;
-    font-size: .875em;
+    .heading-5();
   }
-  .cf-icon {
+
+  .cf-icon-svg {
     display: block;
-    margin: 20px 20px 0;
+    margin: 20px auto 0;
   }
 }
 
+// MINICON SCALE TABLE
+@minicon-sizes: '16px', '20px', '25px', '30px', '40px',
+                '14pt', '21pt', '28pt', '36pt', '48pt';
+
+// Set up looping mixin
+.minicon-scale-loop(@index) when (@index > 0) {
+    // Retrieve the next variable name from the @colors array
+    @size: e(extract(@minicon-sizes, @index));
+    // Output the `.swatch_field__color` rule
+    .minicon-scale-@{size} { font-size: @size; }
+    // Call the next iteration of the loop
+    .minicon-scale-loop((@index - 1));
+}
+
+// Initiate loop
+.minicon-scale-loop( length(@minicon-sizes) );
+
+
+// Asset library
+
 .minicon-asset-lib {
   h3 {
-      margin-top: 1.5em;
+    margin-top: 1.5em;
   }
+
   .minicon-assets-div {
```

Comment:
`.minicon-assets-div`s no longer exist. All this Less can get chucked.

Reply:
I had thought the same thing but turns out it does for the spinning icon in the Animation section. I'd like us to spend some time reworking this page, but for now I think we should leave it.